package.json

```
@@ -43,7 +43,7 @@
     "sinon-chai": "^3.3.0"
   },
   "engines": {
-    "node": ">= 8"
+    "node": ">= 12"
```

```suggestion
    "node": "12.* || 14.* || >= 16"
```

Node 13 and 15 are unsupported already

---

We should have a semver string like this handy every time we have to roll through a bunch of projects and update. Maybe even a codemod?

---

You could do something like https://github.com/kellyselden/standard-node-template/blob/master/package.json#L27 and then it will get auto updated everywhere like https://github.com/kellyselden/eslint-plugin-json-files/pull/84.